frontend/src/shared/modules/filters/components/filterTypes/MultiSelectAsyncFilter.vue

```diff
@@ -33,6 +33,12 @@
               <span class="el-checkbox__inner" />
             </span>
           </div>
+          <template v-if="config.id === 'organizations'">
+            <span class="border rounded-md p-1 mr-2">
```

Comment:
Can you add `w-6 h-6 flex items-center justify-center` to the container? This will set the correct width and height to it and make sure that the image or icon are centered